_data/tools.yml

```diff
@@ -56,9 +56,7 @@ Poretools: toolshed.g2.bx.psu.edu/repos/iuc/poretools_qualdist/poretools_qualdis
 QED: toolshed.g2.bx.psu.edu/repos/bgruening/silicos_it/ctb_silicos_qed
 RCAS: toolshed.g2.bx.psu.edu/repos/rnateam/rcas/rcas
 RNA STAR: toolshed.g2.bx.psu.edu/repos/iuc/rgrnastar/rna_star
-RNA STAR Solo: toolshed.g2.bx.psu.edu/repos/iuc/rna_starsolo
-rna_star: toolshed.g2.bx.psu.edu/repos/iuc/rgrnastar/rna_star
```

Comment:
please be careful with things that you remove, this can be used on other sites, e.g. rna.usegalaxy.eu

Reply:
Yep, so I grepped for anything matching `id="rna_` and only the single cell tools used that, so I kept only those which had `id="RNA ` ids (which the singlecell, rna, and virology portals use)

Line59 appeared to a be duplicate of Line61, so I deleted it